change/@fluentui-react-swatch-picker-e5e27fd6-7f1e-444e-b5bf-146b6cc0e925.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33274&commitId=4c5536d33c07dd563e83af577ae8a495405096cc&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|44| Changed|
|Avatar Converged.size+image+badge.chromium.png|1| Changed|
|Avatar Converged.badgeMask.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4933| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|2549| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>